src/components/profileEdit/AddressModal.tsx

```
@@ -10,9 +10,27 @@ interface AddressModalProps {
   closeOnOutsideClick?: boolean
 }
 
+// Cypress 환경 여부 판단
+const isCypress = typeof window !== 'undefined' && window.Cypress;
+
+function MockDaumPostcode({ onComplete }: { onComplete: (data: any) => void }) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)